core/common/src/thrift/meta_master.thrift

```diff
@@ -36,9 +36,17 @@ struct MasterInfo {
  8: list<string> zookeeperAddresses // Null means zookeeper is not enabled
 }
 
-struct ExportJournalTOptions {}
+struct ExportJournalTOptions {
+ // The directory to export to within the root UFS.
+ 1: string targetDirectory
+ // Whether to write to the local filesystem instead of the root UFS.
+ 2: bool localFileSystem
+}
 struct ExportJournalTResponse {
  1: string backupUri
+ // If we export to local filesystem, this field will be populated with the hostname of the host we
```

Comment:
Good catch, it will contain the hostname whether or not the uri was local. I've updated to
```
 // The hostname of the master which wrote the journal backup. This is useful
 // when the backup was written to local disk of that host.
```